src/vm/node_modules/VM.js

```diff
@@ -11734,7 +11747,8 @@ function sendDataset(target, dataset, log, callback)
                     }
                 );
             }, function (cb) {
-                zfs(['snapshot', dataset + '@sending'], log,
+                // Use -r for the snapshot, in case of BHYVE datasets.
```

Comment:
Instead:

```
// Use -r for the snapshot in case of any child datasets (bhyve, delegeated datasets, etc)
```

Reply:
Hadn't run delegated datasets when I wrote this comment.  Will fix.  Please mark this resolved when it is to your satisfaction.